content-webapp/src/main/webapp/news-activity-composer-app/components/ContentRichEditor.vue

```diff
@@ -57,12 +58,15 @@
       :posting-news="postingNews"
       :news-id="articleId"
       :news-type="articleType"
+      :space-id="spaceId"
       @post-article="postArticle" />
   </v-app>
 </template>
 
 <script>
 
+import {getURLQueryParam} from '../main';
```

Comment:
can we avoid this import ?

Reply:
no we need this to initialize the data from the url!